python-lib/dku_google/clusters.py

```
@@ -110,7 +110,7 @@ def with_spot_vms(self, use_spot_vms):
 
     def with_nodepool_labels(self, nodepool_labels=[]):
         if nodepool_labels:
-            nodepool_labels_dict = {l["from"]: l["to"] for l in nodepool_labels}
+            nodepool_labels_dict = {label["from"]: label.get("to", "") for label in nodepool_labels}
```

do we want to raise an error explicitly if the label key (from) is empty?